website/apps/transcripts/views.py

```diff
@@ -330,7 +331,7 @@ def get_context_data(self, phase_number='1'):
             'act': selected_act,
         }
 
-class ErrorView(TemplateView):
+class ErrorView(MemorialMixin, TemplateView):
```

Comment:
This isn't actually used anywhere right now. (404 will be handled by Django automatically, although we don't have a template for it…and indeed right now it seems to 500.)